src/Week1/HW01Tests.hs

```diff
@@ -0,0 +1,59 @@
+-- CIS 194, Spring 2015
+--
+-- Test cases for HW 01
+
+module Week1.HW01Tests where
+
+import Week1.HW01
+import Week1.Testing
+
+-- Exercise 1 -----------------------------------------
+
+testLastDigit :: (Integer, Integer) -> Bool
+testLastDigit (n, d) = lastDigit n == d
+
+testDropLastDigit :: (Integer, Integer) -> Bool
+testDropLastDigit (n, d) = dropLastDigit n == d
+
+ex1Tests :: [Test]
+ex1Tests = [ Test "lastDigit test" testLastDigit
+             [(123, 3), (1234, 4), (5, 5), (10, 0), (0, 0)]
+           , Test "dropLastDigit test" testDropLastDigit
+             [(123, 12), (1234, 123), (5, 0), (10, 1), (0,0)]
+           ]
+
+-- Exercise 2 -----------------------------------------
+
+ex2Tests :: [Test]
+ex2Tests = []
```

Comment:
Please add at least 1 or 2 tests for each of the exercise functions.
Then try to load the test module to ghci and run allTests (you need to figure out how to do it using functions from  Week1.Testing module).